docs/GettingStartedDocs/install_oe_sdk-Simulation.md

```diff
@@ -0,0 +1,27 @@
+# Install the Open Enclave SDK (non-SGX Ubuntu 16.04/18.04)
+
+## Considerations
+The Open Enclave SDK can be installed on non-SGX Ubuntu 16.04 and 18.04 systems. It can also be used in simulation mode, although some features may not work. Specifically these features are not supported when running in simulation mode:
+- Enclave signature/attribute verification
+- Data Sealing
+- Attestation (both remote and local)
+- Signal handling
```

Comment:
Would be good to mention that this refers to oe_add_vectored_exception_handler/oe_remove_vectored_exception_handler APIs in enclave.h

Reply:
Not sure if we explicitly want to call out the APIs in the installation SDK document. My suggestion is to rewrite as follows and not reference signal handling or any APIs. 

Exception handling: 
Software exceptions work in simulation mode but not hardware exceptions, such as cpuid as these instructions will not raise an hardware exception in simulation mode.

Reply:
@anitagov hardware exceptions will still _work_ in simulation mode. A SEGV will still be raised; but the registered signal handler on the enclave side will not be called (actually we need to double-check whether we support that; but it is better to underpromise since we aren't yet sure about the direction we want to take the simulation mode)

Reply:
We need to convey that the illegal instructions within an enclave (_SIGILL exception signal is generated on Linux_), such as CPUID, GETSEC, etc. will not generate a #UD fault in simulation mode as the Intel SGX hardware is not exercised.  Handlers can be added with oe_add_vectored_exception_handlers but the handler will not get invoked in simulation mode. Hence, my suggestion to not mention these APIs.